src/banman.h

```diff
@@ -68,6 +77,7 @@ class BanMan
     CClientUIInterface* m_client_interface = nullptr;
     CBanDB m_ban_db;
     const int64_t m_default_ban_time;
+    CRollingBloomFilter m_discouraged{50000, 0.000001};
```

Comment:
Should this have a `GUARDED_BY(m_cs_banned)` lock annotation?

Reply:
Done.